torchft/_test_utils.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
```

Comment:
Could you add some docstrings explaining what these functions do? It's not obvious from just reading them and if they're in a utils file we would want to reuse them where possible.